src/components/MovieInfo/MovieInfo.css

```diff
@@ -23,49 +23,61 @@
   margin-top: 30px;
   filter: brightness(80%);
   border-radius: 10px;
-
-  /*background: white;*/
-  /* rgba( 96, 243, 51, 0.10 ); */
-  box-shadow: 0 8px 32px 0 rgba( 31, 38, 135, 0.37 );
+  box-shadow: 0 0 6px 2px rgb(111, 212, 46);
   backdrop-filter: blur(7.0px);
   border: 1px solid rgba( 255, 255, 255, 0.18 );
 
 }
 
 .movie-stats-container {
   display: flex;
-  flex-wrap: wrap;
-  justify-content: space-between;
+  /*flex-wrap: wrap;*/
+  /*justify-content: space-between;*/
   width: 900px;
   flex-wrap: nowrap;
   align-items: center;
+  text-shadow: 1px 1px 2px black;
   /* width: 23%;
   height: 23; */
 }
 
 .backdrop-img {
-  width: 200px;
+  width: 270px;
   /* opacity: 69%; */
 }
 
 
-.movie-stats{
+.movie-stats {
   /* position: absolute; */
   /* display: flex; */
   /* justify-content: center; */
-  bottom: 20px;
+  /*bottom: 20px;*/
   right: 20px;
   color: #6FD42E;
   font-weight: bolder;
-  font-size: 2.5em;
+  /*font-size: 2.5em;*/
   padding-left: 20px;
   padding-right: 20px;
+  /*padding-bottom: 47px;*/
 }
 
 .movie-title {
   width:auto;
+  font-size: 2.9em;
+}
+
+.release-genre-run, .movie-rating{
+  font-size: 1.5em;
 }
 
-.release-genre-run {
-  font-size: 20px;
+.movie-overview {
+/*  bottom: 20px;*/
+/*  right: 20px;*/
+/*  color: #6FD42E;*/
+/*  font-weight: bolder;*/
+  font-size: 1.3em;
+/*  padding-left: 511px;*/
+/*  color: #6FD42E;*/
+/*  text-shadow: 1px 1px 2px black;*/
+/*  padding-right: 20px;*/
```

Comment:
we can definitely go through these commented out rows to see what we can keep/trash